File: JavaScript/002 - Even Fibonacci numbers/solution.js
function execute() {
    var sum = 0;

    var before = 1;
    var now = 2;
    while (now < 4000000) {        
        if (now % 2 == 0) {
            sum += now;
        }
        now += before;        
        before = now - before;
    }
    return sum;
}

module.exports.execute = execute;

File: GoLang/002 - Even Fibonacci numbers/solution.go
package project_euler

func execute002() int {
	sum := 0

	before := 1
	now := 2
	for now < 4000000 {
		if now%2 == 0 {
			sum += now
		}
		now += before
		before = now - before
	}
	return sum
}
